src/AzureExtension/DataModel/DataObjects/Identity.cs

```diff
@@ -42,6 +42,8 @@ public class Identity
 
     public string Avatar { get; set; } = string.Empty;
 
+    public long IsDeveloper { get; set; } = DataStore.NoForeignKey;
```

Comment:
Suggest putting a comment here to explain that "developer" is the term for "person who opened the PR" since I only knew that from your review comment below.

Reply:
Added comments